components/ILIAS/Mail/classes/class.ilMailTemplateContextService.php

```diff
@@ -105,7 +105,7 @@ public static function getTemplateContexts(?array $a_id = null): array
         if (is_array($a_id) && count($a_id)) {
             $where[] = $DIC->database()->in('id', $a_id, false, 'text');
         }
-        if ($where !== []) {
+        if (count($where)) {
```

Comment:
I suggest to leave this as it is, since this is not related to the actual change, right?